test/byContract/IexecEscrow/IexecEscrowToken.test.ts

```diff
@@ -0,0 +1,481 @@
+                .to.emit(iexecPoco, 'Transfer')
+                .withArgs(AddressZero, accountA.address, 0);
+        });
+        it('Should not deposit tokens when caller is address 0', async () => {
+            const AddressZeroSigner = await ethers.getImpersonatedSigner(AddressZero);
+            await rlcInstance
+                .connect(iexecAdmin)
+                .transfer(AddressZeroSigner.address, standardAmount)
+                .then((tx) => tx.wait());
+            // send some gas token
+            iexecAdmin
+                .sendTransaction({
+                    to: AddressZeroSigner.address,
+                    value: AmountWithDecimals(BigNumber.from(100_000)),
+                })
+                .then((tx) => tx.wait());
+
+            await rlcInstance
+                .connect(AddressZeroSigner)
+                .approve(iexecPoco.address, standardAmount)
+                .then((tx) => tx.wait());
+
+            await expect(
+                iexecPoco.connect(AddressZeroSigner).deposit(standardAmount),
+            ).to.be.revertedWith('ERC20: mint to the zero address');
+        });
+    });
+
+    describe('Deposit for', () => {
+        it('Should deposit tokens for another account', async () => {
+            await rlcInstanceAsAccountA
+                .approve(iexecPoco.address, standardAmount)
+                .then((tx) => tx.wait());
+
+            const depositForParams = {
+                amount: standardAmount,
+                target: accountB.address,
+            };
+            const depositForArgs = Object.values(depositForParams) as [BigNumber, string];
+
+            expect(await iexecPocoAsAccountA.callStatic.depositFor(...depositForArgs)).to.be.true;
+            await expect(iexecPocoAsAccountA.depositFor(...depositForArgs))
+                .to.changeTokenBalances(
+                    rlcInstance,
+                    [accountA, iexecPoco],
+                    [-depositForParams.amount, depositForParams.amount],
+                )
+                .to.emit(rlcInstance, 'Transfer')
+                .withArgs(accountA.address, iexecPoco, depositForParams.amount)
+                .to.changeTokenBalances(iexecPoco, [depositForParams.target], [standardAmount])
+                .to.emit(iexecPoco, 'Transfer')
+                .withArgs(AddressZero, depositForParams.target, depositForParams.amount);
+        });
+        it('Should not deposit tokens for zero address', async () => {
+            await rlcInstanceAsAccountA
+                .approve(iexecPoco.address, standardAmount)
+                .then((tx) => tx.wait());
+
+            const depositForParams = {
+                amount: standardAmount,
+                target: AddressZero,
+            };
+            const depositForArgs = Object.values(depositForParams) as [BigNumber, string];
+            await expect(iexecPocoAsAccountA.depositFor(...depositForArgs)).to.be.revertedWith(
```

Comment:
```suggestion
            await expect(iexecPocoAsAccountA.depositFor(standardAmount, AddressZero)).to.be.revertedWith(
```

Reply:
[remove params and args variables when reverted when possible](https://github.com/iExecBlockchainComputing/PoCo/pull/141/commits/a331a0d25aa3ca3adf68803e4e49c39598630619)